frontend/src/App.tsx

```diff
@@ -1,35 +1,10 @@
-import axios from "axios";
-import { useState } from "react";
-const serverURL = 'http://localhost:8000'
+import {LoginForm} from './Components/loginForm'
+import './index.css';
 
 function App() {
-  const [message, setMessage] = useState<string>("");
-
-  const resetMessage = () => {
-    setMessage("")
-  }
-
-  const handleClick = async () => {
-    const isDisplayingMessage = !!message
-    if (isDisplayingMessage) {
-      resetMessage()
-      return
-    };
-
-    try {
-      const res = await axios.get(`${serverURL}/api`)
-      const serverMessage = res.data.data
-      setMessage(serverMessage)
-    } catch (error) {
-      //error has type 'unknown'. My fix: assert error as any.
-      console.error("Error connecting to backend:", (error as any).message)
-      alert(`Error connecting to backend: ${(error as any).message}`)
-    }
-  }
   return (
     <div className="App">
-      <h2>{message ?? ""}</h2>
-      <button onClick={handleClick}>click me!</button>
+      <LoginForm />
```

Comment:
Something to think about because it'll probably cause a merge conflict with Jess anyways is.

How do we determine which page should show on which URL? Hint: React Router is the package we'll probably be using.

Reply:
I just did some reading on React Router and I get the concept of it but not sure where I should start to avoid the merge conflicts.

Reply:
No need to avoid the conflicts, just when your PR merges or Jess's be ready to resolve conflicts!